### ISSUE TITLE
Improvements to Datatypes

### ISSUE THREAD

**Issue by beckyjackson** [ontodev/valve.py#32]
See discussion - https://github.com/ontodev/valve/issues/4

Datatypes should only allow alphabetic characters and dashes and underscores. This could help us make the grammar less ambiguous; datatypes will never have to be enclosed in quotes, whereas string arguments should always be in double quotes.

Furthermore, we should allow a list of comma-separated datatype names in the parent column.

**Comment by jamesaoverton** [comment]
Yes: alphanumeric, dashes, underscores. Maybe it can't start with a number. We do want to exclude periods.

I prefer space separated lists but maybe I'm warped by Clojure.

**Comment by beckyjackson** [comment]
I'm OK with space separated too, since the datatypes can't have spaces anyway. My default is usually pipe- or comma-separated.
